hopsworks-rest-utils/src/main/java/io/hops/hopsworks/restutils/RESTCodes.java

```diff
@@ -634,7 +634,8 @@ public int getRange() {
       Response.Status.METHOD_NOT_ALLOWED),
     SCHEMA_VERSION_NOT_FOUND(18, "Specified version of the schema not found", Response.Status.NOT_FOUND),
     PROJECT_IS_NOT_THE_OWNER_OF_THE_TOPIC(19, "Specified project is not the owner of the topic",
-      Response.Status.BAD_REQUEST);
+      Response.Status.BAD_REQUEST),
```

Comment:
@kai-chi When is this used?

Reply:
it's used when one tries to do some operations on a topic that doesn't belong to a project